vecmath/include/vecmath/vec.h

```diff
@@ -457,6 +457,24 @@ namespace vm {
             static_assert(S > 3);
             return vec<T,4>(x(), y(), z(), w());
         }
+
+        /**
+        * Adds the given range of vertices to the given output iterator.
+        *
+        * @tparam I the range iterator type
+        * @tparam O the output iterator type
+        * @param cur the range start
+        * @param end the range end
+        * @param out the output iterator
+        */
+        template <typename I, typename O>
```

Comment:
The reason is just so that I can call getVertices generically from VertexToolBase, which is templated on the handle type. Both segment and polygon have a static method getVertices that takes range of either segments or polygons and then inserts all adjacent vertices into another container. By adding a static getVertices to vec as well, VertexToolBase<H> can simply call H::getVertices on its selected handles.

Reply:
I see, this makes sense.